raiden/transfer/mediated_transfer/events.py

```diff
@@ -84,9 +87,9 @@ def from_dict(cls, data: Dict[str, Any]) -> "StoreMessageEvent":
             payment_id=int(data["payment_id"]),
             message_id=int(data["message_id"]),
             message_order=int(data["message_order"]),
-            message=None,
-            is_signed=None,
-            message_type=None
+            message=data["message"],
+            is_signed=data["is_signed"],
+            message_type=data["message_type"]
```

Comment:
We should convert to the `LightClientProtocolMessageType ` enum type, otherwise the object is instanciated with an string.